apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -218,14 +219,26 @@ start_blocked_second(Config) ->
     [ Dev1, Dev2 | _ ] = proplists:get_value(devs, Config),
     N1 = aecore_suite_utils:node_name(Dev1),
     N2 = aecore_suite_utils:node_name(Dev2),
-    T0 = os:timestamp(),
     aecore_suite_utils:start_node(Dev2, Config),
     aecore_suite_utils:connect(N2),
     timer:sleep(2000),
-    ct:log("Peers on dev1: ~p", [rpc:call(N1, aec_peers, all, [], 5000)]),
-    ct:log("Unblocked peers on dev1: ~p",
-           [rpc:call(N1, aec_peers, get_random, [10], 5000)]),
-    await_sync_abort(T0, [N1, N2]).
+
+    %% Check that there is only one non-blocked peer (dev3) and no connected peers
```

Comment:
s/dev3/dev2 ?

Reply:
No, dev2 is blocked so dev3 is the non-blocked one!?